code/modules/power/supermatter/supermatter.dm

```diff
@@ -1147,11 +1149,13 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 				message_admins("[src] has consumed [consumed_object], [suspicion] [ADMIN_JMP(src)].")
 			investigate_log("has consumed [consumed_object] - [suspicion].", INVESTIGATE_SUPERMATTER)
 		qdel(consumed_object)
-	if(!iseffect(consumed_object) && power_changes)
-		matter_power += 200
+	if(!iseffect(consumed_object) && isitem(consumed_object) && power_changes)
+		var/obj/item/consumed_item = consumed_object
+		object_size = consumed_item.w_class
+		matter_power += 70 * object_size
```

Comment:
I'm interested in the conversion you've done from the old numbers to the new. It seems to me that this will make normal items stronger then currently

Reply:
Normal items will increase the matter power by 210, what I did is divide 200 by the number for normal weight (3), that's equal to 66.66 and I rounded it up to 70.
Same for mobs but that is easier to check.
Unless I did a mistake and saw the wrong index for normal weight